light-clients/ethereum-light-client/src/client.rs

```diff
@@ -498,33 +492,33 @@ pub fn do_verify_membership(
 // the counterparty (ethereum in this case) stores it as raw bytes. this will no longer be
 // required with ibc-go v9.
 pub fn canonicalize_stored_value(
-    path: String,
+    _path: String,
     raw_value: Vec<u8>,
 ) -> Result<Vec<u8>, CanonicalizeStoredValueError> {
-    let path = path
-        .parse::<Path>()
-        .map_err(|_| CanonicalizeStoredValueError::UnknownIbcPath(path))?;
-
-    let canonical_value = match path {
-        // proto(any<cometbls>) -> ethabi(cometbls)
-        Path::ClientState(_) => {
-            Any::<cometbls::client_state::ClientState>::decode_as::<Proto>(raw_value.as_ref())
-                .map_err(CanonicalizeStoredValueError::CometblsClientStateDecode)?
-                .0
-                .encode_as::<EthAbi>()
-        }
-        // proto(any<wasm<cometbls>>) -> ethabi(cometbls)
-        Path::ClientConsensusState(_) => Any::<
-            wasm::consensus_state::ConsensusState<cometbls::consensus_state::ConsensusState>,
-        >::decode_as::<Proto>(raw_value.as_ref())
-        .map_err(CanonicalizeStoredValueError::CometblsConsensusStateDecode)?
-        .0
-        .data
-        .encode_as::<EthAbi>(),
-        _ => raw_value,
-    };
-
-    Ok(canonical_value)
+    // let path = path
```

Comment:
Can we create an issue to fix the eth client. Let's first fix it and make sure it works, then we make a common lib for evm libs.

Reply:
https://github.com/unionlabs/union/issues/3127